src/app/modules/group/pages/user/user.component.ts

```diff
@@ -24,7 +24,7 @@ export class UserComponent implements OnInit, OnDestroy {
 
   private refresh$ = new Subject<void>();
   readonly state$ = this.route.params.pipe(
-    switchMap(({ id }) => this.getUserService.getForId(id)),
+    switchMap(({ id }) => this.getUserService.getForId(id as string)),
```

Comment:
You force-cast to string something which is not necessarily a string!

Reply:
fixed
Since it comes from route params, id is necessarily a string … or undefined. But typed as any :shrug: by angular